springfox-spring-web/src/main/java/springfox/documentation/spring/web/readers/operation/.gitignore

```diff
@@ -0,0 +1 @@
+/OperationResponseClassReader.java
```

Comment:
???

Reply:
Something wrong, I accidentally ignored.... I will fix it

Reply:
Fixed!